dask/dataframe/tests/test_dataframe.py

```diff
@@ -4062,6 +4062,14 @@ def test_map_partitions_delays_lists():
     assert any(str(L) == str(v) for v in out.__dask_graph__().values())
 
 
+def test_str_noexpand():
+    s = pd.Series(["a b c d", "aa bb cc dd", "aaa bbb ccc dddd"])
```

Comment:
Can you add `name='foo'` here too? Ensure that it stays.

Reply:
Done